File: PointOfSalesV2/PointOfSalesV2.Entities/Warehouse.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PointOfSalesV2.Entities
{
    public class Warehouse : CommonData
    {
        public long BranchOfficeId { get; set; }

        public string Code { get; set; }

        public string Name { get; set; }

        public List<Inventory> Inventory { get; set; }
        [ForeignKey("BranchOfficeId")]
        public BranchOffice BranchOffice { get; set; }
    }
}

File: PointOfSalesV2/PointOfSalesV2.FrontEnd/ClientApp/src/app/@core/data/warehouse.ts
import { BaseModel } from './baseModel';
import { BranchOffice } from './branchOffice';


export class Warehouse extends BaseModel {

     branchOfficeId :number;
     code:string; 
     name :string;
     inventory :any[];
     branchOffice :BranchOffice; 
     
}